.changeset/new-kids-itch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@preconstruct/cli": minor
+"@preconstruct/eslint-plugin-format-js-tag": minor
+---
+
+`package.json#exports` have been added to limit what (and how) might be imported from the package. Everything that was meant to be exported for public usage stays exported.
```

Comment:
```suggestion
`package.json#exports` have been added to limit what (and how) code might be imported from the package.
```

Removed the second thing since I don't really want to emphasise the `package.json#exports`. Really the `package.json#exports` for `@preconstruct/cli` should be `null` or something since it shouldn't really be exported, it should only be accessible via the CLI but it's just easier to not have it like that.